lib/one_model.ex

```diff
@@ -219,32 +219,39 @@ defmodule OneModel do
         {expressions, opts} = Keyword.split(opts, ~w(select order_by preload limit)a)
```

Comment:
This isn't part of the changes, but does the `a` at the end of the line turn these from strings to atoms? I'm trying to figure out how the expression fields are accessed with an atom in lines 222-226.

I'm only seeing Google showing results for String.to_atom() and some other pattern match way that I haven't seen before.

Reply:
`~w(...)` creates a list of strings. `~w(...)a` creates a list of atoms. So this line is pulling out the list of atoms and their respective values from the `opts` keyword list parameter, then binding them to `expressions` and the leftover from `opts` is bound to `opts`.